build/docker/edgeimg-make.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+set -e
+
+REPOSITORY=${REPOSITORY:-"kubeedge/edgecore"}
+RELEASE_VERSION=$(git describe --tags `git rev-list --tags --max-count=1`)
```

Comment:
```suggestion
RELEASE_VERSION=$(git describe --tags)
```
Use `git describe --tags` should be more accurate, which reflects the latest commit situation.